Content.Server.Database/Model.cs

```diff
@@ -387,4 +389,37 @@ public class ServerBanHit
         public ServerBan Ban { get; set; } = null!;
         public ConnectionLog Connection { get; set; } = null!;
     }
+
+    [Table("server_role_ban")]
+    public sealed class ServerRoleBan
+    {
+        public int Id { get; set; }
+        public Guid? UserId { get; set; }
+        [Column(TypeName = "inet")] public (IPAddress, int)? Address { get; set; }
+        public byte[]? HWId { get; set; }
```

Comment:
Having HWID/IP matching in role bans doesn't make much sense to me. Sure, this would avoid "make new account to evade role bans" but it feels like there should be better ways to track that because job bans are not the only thing we would want to match for people with a "criminal" record (since we should not allow alting in the first place).

It also makes the code for this much more complex.

Reply:
I personally asked for this, and it's not for handling alts (well hwid is but)
It's for banning households from playing important roles to reduce metagame potential.

Reply:
imo it's perfectly fine to make all the different types of ban equivalent features wise.